recipes/libattr/all/conanfile.py

```diff
@@ -0,0 +1,111 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibAttrConan(ConanFile):
+    name = "libattr"
+    description = "Commands for Manipulating Filesystem Extended Attributes"
+    topics = ("conan", "attr", "filesystem")
+    license = "GPL-2.0-or-later"
+    homepage = "https://savannah.nongnu.org/projects/attr/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_folder(self):
+        return "build"
+
+    @property
+    def _pkg_etc(self):
+        return os.path.join(
+            self.package_folder,
+            "etc"
+        )
+
+    @property
+    def _pkg_lib(self):
+        return os.path.join(
+            self.package_folder,
+            "lib"
+        )
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def config_options(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "libattr is just supported for Linux")
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(
+            self, win_bash=tools.os_info.is_windows)
+        conf_args = [
+            "--prefix={}".format(tools.unix_path(self.package_folder)),
+            "--bindir={}".format(tools.unix_path(
+                os.path.join(self.package_folder, "bin"))),
+            "--libdir={}".format(tools.unix_path(
+                os.path.join(self.package_folder, "lib")))
+        ]
+        if self.options.shared:
+            conf_args.extend(["--enable-shared", "--disable-static"])
+        else:
+            conf_args.extend(["--disable-shared", "--enable-static"])
+
+        self._autotools.configure(
+            args=conf_args, use_default_install_dirs=False)
```

Comment:
This is the first recipe on cci that would need to do `use_default_install_dirs=False`.
What error is emitted when you would not pass this arguments and not add `--prefix/--bindir/--libdir` to the `conf_args` list?